tests/datasets/test_bigearthnet.py

```diff
@@ -148,3 +149,18 @@ def test_not_downloaded(self, tmp_path: Path) -> None:
         "to automaticaly download the dataset."
         with pytest.raises(RuntimeError, match=err):
             BigEarthNet(str(tmp_path))
+
+    def test_plot(self, dataset: BigEarthNet) -> None:
```

Comment:
Hmm, this dataset actually necessitates testing both success and failure in the same function? Now I'm starting to doubt my advice on the previous dataset...

Reply:
yeah when you do have a fixture you will need to do it this way, or split it into multiple fixtures, or make plot() work with non-rgb